static/js/ranking/services/ranking.service.js

```diff
@@ -6,7 +6,7 @@
   'use strict';
 
   angular
-    .module('crowdsource.ranking.services')
+    .module('crowdsource')
```

Comment:
I just checked it, you're right, it works under crowdsource.ranking.services . Do you want me to shift the service back to crowdsource.ranking.services ?